src/components/buttons/button-accessibility-image-alt.jsx

```diff
@@ -20,29 +20,23 @@ class ButtonAccessibilityImageAlt extends React.Component {
 
 	static key = 'imageAlt';
 
-	constructor(props) {
-		super(props);
-
-		const editor = props.context.editor;
-
+	componentDidMount() {
+		const editor = this.props.context.editor;
```

Comment:
As per our Slack chat, if this is not a timing issue, and moving the code from the `constructor` to `componentDidMount` doesn't change the behavior, then there is no point in moving it. Is there any reason why you couldn't make this same change (calling `getStartElement()`)  in-place where the code used to be?

As it is currently written, you are reassigning to `this.state` in `componentDidMount()`, which is wrong. The only legit place to assign like that is in the `constructor`. Elsewhere it has to be via `this.setState`, and even then it is not allowed in all places:

![1799171_10201997263550081_674283271_o](https://user-images.githubusercontent.com/7074/76536036-14729f80-647c-11ea-95da-dabb09102a03.jpg)